src/main/java/gui/RootCenterArea.java

```diff
@@ -85,7 +85,8 @@ private void initMainTimeLinePane() {
         // main timeline panes
         mainTimelineScrollpane = new ScrollPane();
         mainTimeLineAnchorPane = new AnchorPane();
-        mainTimeLineGridPane = new TimelinesGridPane(numberOfTimelines, numberOfCounts, 1000, 1000);
```

Comment:
I think you should use constants here and the following 3/4 methods on the 10's, 100's and 1000's as stated in previous comments